test/test_ssl.rb

```diff
@@ -206,8 +206,20 @@ def test_sysread_and_syswrite
     }
   end
 
+  def test_sysread_nonblock_and_syswrite_nonblock_keywords
```

Comment:
@jeremyevans do you mind letting me know what the point of this test was and how we can make it work more reliably?

Reply:
Also, do we expose `sysread_nonblock` and `syswrite_nonblock` because those seem counter intuitive.